assets/src/application/ui/display/EmptyState.tsx

```diff
@@ -2,31 +2,31 @@
  * External dependencies
  */
 import * as React from 'react';
+import { __ } from '@wordpress/i18n';
 import { Empty } from 'antd';
 
 interface EmptyStateProps {
-	children: React.ReactChild;
+	children?: React.ReactNode;
+	className?: string;
 	description: string;
-	title: string;
+	title?: string;
 }
 
-const EmptyState = (props: EmptyStateProps) => {
-	const { children, description, title } = props;
-
+const EmptyState = ({ className, children, description, title }: EmptyStateProps) => {
 	return (
-		<>
+		<div className={className}>
 			<Empty
 				description={
 					<>
-						{title}
-						<span>{description}</span>
+						{title && __(title)}
+						<span>{__(description)}</span>
```

Comment:
couple of points:

- I don't think you can run translations on variables like this (although I could be wrong), so let's just assume the incoming strings were translated at the source (which I think should be the case, otherwise it's a guessing game as to whether the receiving component has handled translations.)

- we should account for `description` being blank like you did with `title` no?

so this should be (imho) something like:

```js
{title ? title : '' }
{description ? <span>description</span> : '' }
```

I used ternarys because `{title && title}` just doesn't look right to me, but maybe it's ok?

Reply:
`{title && title}` is just short circuit evaluation I have often seen in react apps
updated: https://github.com/eventespresso/event-espresso-core/pull/2366/commits/85c7143fc0ec48d09b63c0b33da50cdd3161fbaf